Python/libraries/recognizers-text/recognizers_text/Matcher/string_matcher.py

```diff
@@ -43,8 +46,30 @@ def init(self, values: [], ids: [] = [], values_dictionary: {} = {},
              tokenized_values: [] = []) -> None:
         pass
 
-    def find(self, tokenized_query: [], query_text: str = "") -> []:
-        pass
+    @dispatch(object, object)
+    def find(self, tokenized_query: []) -> []:
+        return self.find(tokenized_query)
```

Comment:
This should not call self, you have a property called matcher.
`self.matcher.find(tokenized_query)`